src/dialog-editor/components/field/field.html

```diff
@@ -43,14 +43,20 @@
     </div>
 
     <!-- drop down list -->
-    <select ng-switch-when="DialogFieldDropDownList"
-            ng-model="vm.fieldData.default_value"
-            class="form-control">
-      <option ng-repeat="option in vm.fieldData.values"
-              value="{{ option[0] }}">
-        {{ option[1] }}
-      </option>
-    </select>
+    <div ng-switch-when="DialogFieldDropDownList">
+      <div ng-if="!vm.modalData.options.force_multi_value">
+        <select class="form-control" pf-select
+                ng-model="vm.modalData.default_value"
+                ng-options="entry as entry[0] for entry in vm.modalData.values">
+        </select>
+      </div>
+      <div ng-if="vm.modalData.options.force_multi_value">
+        <select class="form-control" multiple pf-select
+                ng-model="vm.modalData.default_value"
+                ng-options="entry as entry[0] for entry in vm.modalData.values">
```

Comment:
come on! do we need to c-n-p the whole block to get an extra attribute `multiple`

Reply:
@martinpovolny I was trying to do it in a better way, but couldn't find one 

https://docs.angularjs.org/error/$compile/selmulti?p0=%3Cselect%20class%3D%22form-control%22%20pf-select%3D%22%22%20ng-attr-multiple%3D%22%7B%7Bvm.modalData.options.force_multi_value%20%3F%20true%20:%20false%7D%7D%22%20ng-model%3D%22vm.modalData.default_value%22%20ng-options%3D%22entry%20as%20entry%5B0%5D%20for%20entry%20in%20vm.modalData.values%22%3E%22